conda-forge.yml

```diff
@@ -4,3 +4,6 @@ github:
 conda_build:
   error_overlinking: true
 conda_forge_output_validation: true
+azure:
+  free_disk_space: True
+  upload_packages: True
```

Comment:
Somewhat tangential, but would suggest sorting the top-level keys alphabetical. This is what the migration/update bot does. As a result if these are sorted alphabetically, it will avoid unneeded no-op migration PRs